src/Microsoft.IdentityModel.Tokens/Validation/Validators.IssuerSigningKey.cs

```diff
@@ -40,18 +37,15 @@ public static partial class Validators
         /// <param name="securityToken">The <see cref="SecurityToken"/> being validated.</param>
         /// <param name="validationParameters">The <see cref="ValidationParameters"/> to be used for validating the token.</param>
         /// <param name="configuration">The <see cref="BaseConfiguration"/> to be used for validation.</param>
-        /// <param name="callContext">The <see cref="CallContext"/> to be used for logging.</param>
-        /// <exception cref="SecurityTokenArgumentNullException"> if 'securityKey' is null and ValidateIssuerSigningKey is true.</exception>
-        /// <exception cref="SecurityTokenArgumentNullException"> if 'securityToken' is null and ValidateIssuerSigningKey is true.</exception>
-        /// <exception cref="SecurityTokenArgumentNullException"> if 'validationParameters' is null.</exception>
+        /// <param name="callContext">The <see cref="CallContext"/> that contains call information.</param>
         internal static ValidationResult<ValidatedSigningKeyLifetime> ValidateIssuerSigningKey(
             SecurityKey securityKey,
             SecurityToken securityToken,
             ValidationParameters validationParameters,
 #pragma warning disable CA1801 // Review unused parameters
             BaseConfiguration? configuration,
 #pragma warning restore CA1801 // Review unused parameters
-            CallContext? callContext)
+            CallContext callContext)
```

Comment:
Have we verified that changes like this one won't break any friend assemblies that might be using it?

Reply:
No part of the new validation model has been officially released as of yet. We have not included any piece of this in SAL or MISE (other than Brent experimenting on a branch, maybe).